front/app/containers/Admin/projects/project/general/components/ProjectFolderSelect/ProjectFolderSelect.test.tsx

```diff
@@ -94,4 +102,32 @@ describe('ProjectFolderSelect', () => {
       'disabled'
     );
   });
+  it('should set folder_id to null when "no" option is selected', async () => {
```

Comment:
Following up on your question on why this fails. It looks like the test is still using the same mock from the previous test. `mockPermission`. Setting that to false means that we the user can create a project outside a folder

Reply:
Good guess! Thank you, let me try it.

Reply:
Yes, it works. Thanks a lot, I've already lost hope that I could add this test :)

It doesn't seem right that one test affects the other. Maybe one day we'll fix it.

Reply:
Indeed one test should not affect the other. The best practice is to normally reset / restore the mocks after a test. In this case, we can add a beforeEach that can ensure the default mock for this test.

Something like

```
  beforeEach(() => {
    mockPermission = false;
  });
  ```
  
  This works though so I'll approve and leave it up to you on if you want to change or not